packages/blade/package.json

```diff
@@ -283,7 +284,8 @@
     "react-native-svg": "^12.3.0",
     "react-native-gesture-handler": "^2.9.0",
     "@gorhom/bottom-sheet": "^4.4.6",
-    "@gorhom/portal": "^1.0.14"
+    "@gorhom/portal": "^1.0.14",
+    "@razorpay/i18nify-js": "^1.3.1"
```

Comment:
Is this a mandatory peer dep? Does this work with react-native too? If not, we may have to make it optional under the `peerDependenciesMeta` key.

Reply:
Yes it's a mandatory peer dep and it works with react-native too. 
Check this [section](https://docs.google.com/document/d/1jr4uoI19-qvgohuN1fosv2lSrmINhxXkZVRHzvd9WBo/edit#heading=h.84rlxo6w3k3e) in tech spec for an integration example on react-native.